CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [x] - Next
+
+- Fix alignment issue
+- Fix positioning issues
+- Change width and height to be defined on animations paint time (breaking).
+
+
```

Comment:
One blank line is enough :)

Reply:
The human linter 2, whitespace strikes back.

Reply:
Come on, extra blank lines are annoying :P